pkg/template/event.go

```diff
@@ -92,6 +92,8 @@ func getEventPathValue(event []byte, eventPath string) (string, error) {
 	return eventPathValue, nil
 }
 
+// NewResources returns all resources defined when applying the event to the
+// TriggerTemplate and TriggerBinding in the ResolvedBinding.
```

Comment:
Thanks for adding a descriptive comment to this public function 😄 Shouldn't our linter have caught this?

Reply:
We need to update the extension in our linter. We have an issue tracking this, but I couldn't get it working for some reason.

Reply:
Ah, here it is https://github.com/tektoncd/plumbing/issues/64